plugins/data-management/src/AddTrackWidget/components/AddTrackWidget.tsx

```diff
@@ -86,8 +97,36 @@ function AddTrackWidget({ model }: { model: AddTrackModel }) {
           sequenceAdapter: getConf(assemblyInstance, ['sequence', 'adapter']),
         },
       })
+      const textSearchingDefault = {
+        attributes: ['Name', 'ID'],
+        exclude: ['CDS', 'exon'],
+      }
+      const supported = [
+        'Gff3TabixAdapter',
+        'VcfTabixAdapter',
+        'Gff3Adapter',
+        'VcfAdapter',
+      ]
```

Comment:
does this need to be using the text-indexing supported function instead?

Reply:
May have an issue (issue related to readline in web)  if we add the text indexing package to the data management plugin in order to use the supported method. Could move the supported to core/util?